contrib/apelscripts/condor_batch_blah.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+fail () {
+    echo "$@" >&2
+    exit 1
+}
+
+safe_config_val () {
+    var=$1
+    attr=$2
+    val=$(condor_ce_config_val $attr) ||
+    fail "Failed to retrieve CE configuration value '$attr'"
+    eval "$var"='$val'
+}
+
+OUTPUT_DIR="$(condor_ce_config_val APEL_OUTPUT_DIR)"
+[[ -d $OUTPUT_DIR && -w $OUTPUT_DIR ]] || fail "Cannot write to $OUTPUT_DIR"
+OUTPUT_DATETIME=`date '+%Y%m%d-%H%M'`
+
+HISTORY_DIR=`condor_config_val PER_JOB_HISTORY_DIR`
+QUARANTINE_DIR=$HISTORY_DIR/quarantine
+[[ -d $QUARANTINE_DIR && -w $QUARANTINE_DIR ]] || fail "Cannot write to $QUARANTINE_DIR"
+
+HISTORY_EXTRA_ARGS=(-format "\n" EMPTY)
```

Comment:
Since we're not using `condor_history` anymore, we should probably rename the var to `CONDOR_Q_EXTRA_ARGS` or something